### ISSUE TITLE
Rotate AWS Keys - May 30, 2024

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#76343]
## Summary
We need to rotate the IAM keys in AWS every 60 days. 

## Resources
[Documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/engineering/rotating-iam-keys.md)
[Knowledge sharing meeting recording](https://drive.google.com/file/d/1owR-Pec3urKlulr3ysarHXER2bvbd7Ia/view?ts=65baa17b)

## Tasks
- Rotate the keys

**Comment by ndouglas** [comment]
I don't have the permissions needed for this, nor does @olivereri, so I opened a SNOW ticket to request them for myself.

**Comment by ndouglas** [comment]
Blocked by not having sufficient permissions to make these changes in Datadog. Opened another ticket to get those permissions 🙂

**Comment by ndouglas** [comment]
- [x] `chip` and `lorota` repos updated with `CMS-service-account-1` secrets.
- [x] thread opened to advise vetext team about updates [here](https://dsva.slack.com/archives/C051S52N1C6/p1717094075977969). 
- [ ] tevi datadog needs to be updated with service-account-2 secrets
- [ ] old keys need to be removed when appropriate

**Comment by ndouglas** [comment]
Keys haven't been used for 11 days and 6 days, deactivated and deleted 👍🏻
